9_collaborator_data.yml

```diff
@@ -156,6 +156,21 @@ targets:
       subnet_ind = '9_collaborator_data/umn/network_subset.rds.ind',
       out_ind = target_name)
 
+  9_collaborator_data/umn/obs_temp_subset.zarr.tar.ind:
+    command: write_zarr_tarfile(dat_ind = '9_collaborator_data/umn/obs_temp_subset.csv.ind', out_ind = target_name)
+    
+  9_collaborator_data/umn/obs_temp_full.zarr.tar.ind:
+    command: write_zarr_tarfile(dat_ind = '9_collaborator_data/umn/obs_temp_full.csv.ind', out_ind = target_name)
+
+  9_collaborator_data/umn/obs_flow_subset.zarr.tar.ind:
+    command: write_zarr_tarfile(dat_ind = '9_collaborator_data/umn/obs_flow_subset.csv.ind', out_ind = target_name)
+    
+  9_collaborator_data/umn/obs_flow_full.zarr.tar.ind:
+    command: write_zarr_tarfile(dat_ind = '9_collaborator_data/umn/obs_flow_full.csv.ind', out_ind = target_name)
+
+  9_collaborator_data/umn/uncal_sntemp_input_output.zarr.tar.ind:
```

Comment:
Do we want to create a new folder for these output or is `umn` preferred? I suppose these are all the same files as used for `umn` just a different format so seems appropriate to be here.

Reply:
I agree that `umn` isn't all that descriptive of this. But yeah, my thinking was since they have the same names and just a different format I'd leave them here. Hopefully we can remember that that is where they are at :)

Reply:
yeah I think that's fine to keep in `umn` folder